boards/realtek/rts5912_evb/rts5912_evb.dts

```diff
@@ -0,0 +1,47 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+ *
+ */
+
+/dts-v1/;
+#include <realtek/ec/rts5912.dtsi>
+#include <realtek/ec/rts5912-pinctrl.dtsi>
+#include <zephyr/dt-bindings/gpio/gpio.h>
+/ {
+	model = "Realtek RTS5912 Evaluation Board";
+	compatible = "realtek,rts5912-evb";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,console = &uart0;
+		zephyr,flash = &flash0;
+	};
+
+	power-states {
+		idle: idle {
+			compatible = "zephyr,power-state";
+			power-state-name = "suspend-to-idle";
+			min-residency-us = <10000>;
+		};
+
+		suspend_to_ram: suspend_to_ram {
+			compatible = "zephyr,power-state";
+			power-state-name = "suspend-to-ram";
+			min-residency-us = <250000>;
+		};
+	};
+};
+
+&cpu0 {
+	cpu-power-states = <&idle &suspend_to_ram>;
+};
+
```

Comment:
non-blocking comment: should there be flash partitions here?

Reply:
unaddressed

Reply:
power states must not be at board level

Reply:
We have put those power state to rts5912.dtsi.